src/config/sequelizeConfig.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { SequelizeModuleOptions, SequelizeOptionsFactory } from '@nestjs/sequelize';
+import { Dialect } from 'sequelize';
+import { IDatabaseConfig } from './configuration';
+import { Todo } from '../todos/models/todo.model';
+import { Session } from '../session/models/session.model';
+import { UserSignature } from '../userSignature/models/userSignature.model';
+import { EnumConfig } from './enumConfig/enumConfig';
+
+@Injectable()
+export class SequelizeConfigService implements SequelizeOptionsFactory {
+  constructor(private readonly configService: ConfigService) {}
+
+  createSequelizeOptions(): SequelizeModuleOptions {
+    const {
+      pg: { host, port, username, password, database },
+    } = this.configService.get<IDatabaseConfig>(EnumConfig.DATABASE);
+
+    return {
+      dialect: <Dialect>process.env.SQL_DIALECT || 'postgres',
+      logging: process.env.SQL_LOGGING === 'true' || true,
```

Comment:
This is wrong, you are not allowed to use process.env outside of configs. Move SQL_LOGGING and SQL_DIALECT to config